build.gradle

```diff
@@ -497,7 +497,7 @@ asciidoctorPdf {
 
 	asciidoctorj {
 		sourceDir "$buildDir/asciidoc"
-		inputs.dir(sourceDir)
+		inputs.dir(sourceDir).withPathSensitivity(PathSensitivity.RELATIVE)
```

Comment:
I guess something like `build/asciidoc` would work as well, but then `sourceDir` property of the `asciidoctorj` plugin might not be happy.
I'll merge after your answer.
Thanks

Reply:
> I guess something like `build/asciidoc` would work as well, but then `sourceDir` property of the `asciidoctorj` plugin might not be happy.

Still, the full absolute path will be considered. This is a quirk of Gradle. The [default path sensitivity is absolute](https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/PathSensitive.html).